SUMMARY.md

```diff
@@ -13,6 +13,8 @@
   * [Reputation management](use-cases/reputation-management.md)
   * [Revenue management](use-cases/revenue-management.md)
   * [Upselling](use-cases/upselling.md)
+  * [Payment automation](/use-cases/payment-automation)
```

Comment:
All the other use-cases here are describing business use cases while this is a "horizontal" use-case that is applicable across all verticals that take payments. I propose to think of some better structure. Also, the existing use cases already contain tons of "duplicate" documentation which could be classified as "horizontal". E.g. how to get all reservations and keep them up-to-date. @vitasamek @Jordsips @stang704 can you think of some better structure for this whole part of docs?

Maybe having "applications" section that will represent the verticals (like RMS application, accounting application...). And use-cases being "horizontal". So that e.g. within upselling application, you'll refer to "reservation synchronization" use-case and "payments" use-case, becase upselling app is built using these use-cases.

Reply:
good point - I remember a few months ago with Josef we briefly talked about the idea of organising the use cases by "functionality" rather than by specific apps and have the documentation dynamically suggest to a partner a bespoke use case depending on the functionality they want to build - pretty much like what you're suggesting. 

maybe time to revisit the topic :D